npm-packages/pyoes/scss/pyoes/_oe-forms.scss

```diff
@@ -41,11 +41,30 @@
 .checkbox-custom, .radio-custom { opacity: 0; position: absolute; }
 .checkbox-custom, .checkbox-custom-label, .radio-custom, .radio-custom-label { display: inline-block; vertical-align: middle; margin: 5px; cursor: pointer; font-size:0.8em; }
 .checkbox-custom-label, .radio-custom-label { position: relative; }
-.checkbox-custom + .checkbox-custom-label:before, .radio-custom + .radio-custom-label:before { content: ''; background: $fresh-blue; display: inline-block; vertical-align: middle; width: 20px; height: 20px; padding: 2px; margin-right: 10px; text-align: center; font-size:0.8em; }
-.checkbox-custom:checked + .checkbox-custom-label:before { content: '\2714'; background: $fresh-blue; color: #fff; }
+
+.checkbox-custom + .checkbox-custom-label:before, .radio-custom + .radio-custom-label:before {
+  content: '';
+  background: $fresh-blue;
+  display: inline-block;
+  vertical-align: middle;
+  width: 20px;
+  height: 20px;
+  padding: 2px;
+  margin-right: 10px;
+  text-align: center;
+  font-size: 0.8em;
+  font-family: "FontAwesome";
+}
+.checkbox-custom:checked + .checkbox-custom-label:before { background: $fresh-blue;}
 .radio-custom + .radio-custom-label:before { border-radius: 50%; }
-.radio-custom:checked + .radio-custom-label:before { content: '\2714'; color: #bbb; }
 
+.radio-custom:checked + .radio-custom-label:before, .checkbox-custom:checked + .checkbox-custom-label:before {
+  color: white;
+  content: "\f00c";
+  font-size: 14px;
+  padding: 0;
+  line-height: 20px;
```

Comment:
ik ga ervan uit dat die font size en line height echt nodig zijn? :)

Reply:
De line height al zeker wel. Zonder de font size ziet de check er te klein uit imho, nu ziet het eruit zoals in de beschrijving van het ticket.